packages/datagateway-dataview/cypress/e2e/table/investigations.cy.ts

```diff
@@ -215,6 +217,58 @@ describe('Investigations Table', () => {
     });
   });
 
+  describe('should be able to use text filters', () => {
```

Comment:
A minor point - these tests should probably be in the `should be able to filter by` section, and replace the `text` test with this describe block (as we don't need the existing text filter test as this is tested by the include test case in this new block)

Reply:
Moved the tests to `should be able to filter by` section and removed `text` test.